health_monitor/include/health_monitor.h

```diff
@@ -82,7 +82,7 @@ class HealthMonitor
         {"pressure_sensor", health_monitor::ReportFault::PRESSURE_NODE_DIED},
         {"mission_manager", health_monitor::ReportFault::MISSION_NODE_DIED},
         {"pose_estimator", health_monitor::ReportFault::POSE_NODE_DIED},
-        {"thruster", health_monitor::ReportFault::THRUSTER_NODE_DIED},
+        {"thruster_control_node", health_monitor::ReportFault::THRUSTER_NODE_DIED},
```

Comment:
@patapufate this should be in its own PR. Also, we need to come up with a better way to do this mapping at some point.